packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiClient } from '../.'
+
+/**
+ * Loan RPCs for DeFi Blockchain
+ */
+export class Loan {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Destroys a loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme, max 8 chars
+   * @param {number} [activateAfterBlock] Block height at which new changes take effect
+   * @param {DeleteLoanSchemeOptions} [options]
+   * @param {UTXO[]} [options.utxos = []] Specific UTXOs to spend
+   * @param {string} [options.utxos.txid] Transaction Id
+   * @param {number} [options.utxos.vout] Output number
```

Comment:
```suggestion
   * @param {string} options.utxos.txid Transaction Id
   * @param {number} options.utxos.vout Output number
```
not optional within `UTXO`